app/models/release.rb

```diff
@@ -1,13 +1,17 @@
 # frozen_string_literal: true
 class Release < ActiveRecord::Base
+  NUMBER = '\d+(:?.\d+)*'
```

Comment:
Don't you need a backslash before the `.` in there, so it's treated as a literal period?